overlord/ifacestate/ifacemgr_test.go

```diff
@@ -49,11 +51,12 @@ type interfaceManagerSuite struct {
 	state           *state.State
 	db              *asserts.Database
 	privateMgr      *ifacestate.InterfaceManager
+	privateHookMgr  *hookstate.HookManager
 	extraIfaces     []interfaces.Interface
 	secBackend      *interfaces.TestSecurityBackend
 	restoreBackends func()
-
-	storeSigning *assertstest.StoreStack
+	command         *testutil.MockCmd
```

Comment:
I'd just call this mockSnapCmd and always do it (if that's what you need) so that there are less tests and it is more obvious what's going on. Feel free to ignore though as I'm just nitpicking here based on personal preference.